src/collapse/perf.rs

```diff
@@ -280,7 +280,10 @@ impl Folder {
         let pc = line.next()?.trim_end();
         let mut line = line.next()?.rsplitn(2, ' ');
         let mut module = line.next()?;
-        // module is always wrapped in (), so remove those
+        // module should always be wrapped in (), so remove those
+        if !module.starts_with('(') || !module.ends_with(')') {
```

Comment:
This seems like an unrelated change?

Reply:
When using `inferno-collapse-guess` on a `sample` profile it would panic here when called from `is_applicable`. This is to prevent that.

Reply:
Maybe update the comment to reflect that?